lib/companies_house/client.rb

```diff
@@ -16,6 +16,8 @@ def initialize(config)
       raise ArgumentError, "Missing API key" unless config[:api_key]
       @api_key = config[:api_key]
       @endpoint = URI(config[:endpoint] || ENDPOINT)
+      @open_timeout = config[:open_timeout] || 60
+      @read_timeout = config[:read_timeout] || 60
```

Comment:
These are the default values for `Net::HTTP` - thought it better to be explicit.